apps/examples/e2e/tests/test-licensing.spec.ts

```diff
@@ -8,7 +7,7 @@ async function setupLicenseTest(page: any, licenseKey: string) {
 			window.__TLDRAW_LICENSE_KEY__ = "${licenseKey}";
 		`)
 	await page.goto('http://localhost:5420/end-to-end')
-	await page.waitForTimeout(LICENSE_TIMEOUT + 1000) // Wait for license processing
+	await page.waitForTimeout(6000) // Wait for license processing
```

Comment:
Was there an issue with importing from tldraw?

Reply:
yeah, playwright uses cjs and whatever they're using for transpilation doesn't handle `import.meta` correctly. This won't affect anyone else though - in the esbuild cjs builds we publish the `import.meta` stuff gets transformed into an empty object.